src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -120,7 +120,7 @@ case class Bagger(
       val biasTraining = oobErrors.map { case (f, e, u) =>
         // Math.E is only statistically correct.  It should be actualBags / Nib.transpose(i).count(_ == 0)
         // Or, better yet, filter the bags that don't include the training example
-        val bias = Math.max(Math.abs(e) - u, 0)
+        val bias = Math.max(Math.abs(e) - u * ratio, 0)
```

Comment:
This wasn't contributing the issue, but feels right: since we compute the ratio before training the bias model, we should scale the uncertainty in the bias model's training in the same way that we will rescale it when the model is applied.